apps/docs/src/app/platform/component-docs/platform-forms/platform-form-generator/platform-form-generator-examples/platform-form-generator-custom-error-example.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, ViewChild } from '@angular/core';
+import { Validators } from '@angular/forms';
+import { DynamicFormItem, FormGeneratorComponent, FormGeneratorService } from '@fundamental-ngx/platform';
```

Comment:
please use the appropriate imports path

Reply:
Platform does not support deep imports yet